### PR TITLE
Update in/2024-02/02-05 Inside-Story

### PR DESCRIPTION
**Pull request by PaulTitto**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Have you run `run.sh` or `node deploy.js -b test -l [language] -q [quarter]`?
- [ ] Have you checked for semicolons in the titles?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

For example, `./run.sh -l en -q 2022-01` or `node deploy.js -b test -l en -q 2018-02`.

### CODE REVIEW

**Comment by PaulTitto** [comment]
in "Update in/2024-01/01"  i want test dropdown it's work for bahasa indo or no heheheh. Thxu

File: src/in/2024-01/01/01.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ Kitab Mazmur telah ditulis dalam puisi Ibrani oleh penulis yang berbeda dari zam
 
 _*Pelajari pelajaran pekan ini untuk persiapan Sabat, 6 Januari._
 
-#### Bacaan Tambahan: Kutipan dari Ellen G. White
+---
+
+#### Additional Reading: Selected Quotes from Ellen G. White
```
Comment (VitalikL) [inline src/in/2024-01/01/01.md:22]: @PaulTitto If you want me to make this text "Bacaan Tambahan: Kutipan dari Ellen G. White" collapsable

Comment (PaulTitto) [reply src/in/2024-01/01/01.md:22]: Yes sir, for the Indonesian version the text can be collapsed, Please. Thxu

**Comment by VitalikL** [comment]
@PaulTitto done. Starting this quarter it should be collapsed

**Pull request merged**